pyoxidizer.bzl

```diff
@@ -24,7 +24,7 @@ def make_exe():
     # Obtain the default PythonDistribution for our build target.  We link this
     # distribution into our produced executable and extract the Python standard
     # library from it.
-    python_dist = default_python_distribution()
+    python_dist = default_python_distribution(python_version = "3.8")
```

Comment:
This is not strictly necessary anymore given yesterday's release of certifi 2022.06.15.2, so I'm inclined to drop it.

Reply:
Dropped.